README.md

```diff
@@ -30,16 +30,20 @@ and file location that are the cause of the problem, as well as a link
 to the implementation of that query in the current version of the tool.
 
 Notes:
-- If using it on a massive codebase (multiple hundreds of thousands of lines of Rust),
-  the queries may be a bit slow: there is some `O(n^2)` scaling for `n` items in a few places that
-  I haven't had time to optimize down to `O(n)` yet. Apologies! I have temporarily prioritized
-  features over speed, and the runtime will improve significantly with a small amount of extra work.
 - **No false positives**: Currently, all queries report constructive proof of semver violations:
   there are no false positives. They always list a file name and line number for the baseline item
   that could not be found in the new code.
 - **There are false negatives**: This tool is a work-in-progress, and cannot check all kinds of
   semver violations yet. Just because it doesn't find any semver issues doesn't mean
   they don't exist.
+- **Support for most recent stable and beta Rust**: Different Rust compiler versions
+  produce different rustdoc format versions. The most recent `cargo-semver-checks` always
+  supports the most recent Rust stable and beta versions. It *may* also support some nightly
```

Comment:
> The most recent `cargo-semver-checks` always supports the most recent Rust stable and beta versions.

I'd more carefully qualify this, as this says that you will maintain the adapter until rustdoc-json is stable. I'd think long and hard before publicly committing to that.

Reply:
Could you expand on how you are reading that into it because I'm not.

Reply:
I think it's worth clarifying to the user that keeping up with stable requires ongoing maintenance from the maintainers of `cargo-semver-checks`. This is in contrast all rust crates, where once they work on stable will work on stable without ongoing support.

The way it's written makes it sound like the MSRV is the latest rust, and that you will drop support for old stables immediately, which is much more common in the ecosystem.